labels/en-US.json

```diff
@@ -484,6 +484,7 @@
   "order": "Order",
   "orderCancelBlob1": "The following item(s) from order #{0}, external order #{1} have been cancelled.",
```

Comment:
Not part of your story but we are missing translation here..! orderCancelBlob1 and orderCancelBlob2

Reply:
Done.